src/fshtypes/ValueSet.ts

```diff
@@ -7,15 +7,13 @@ import { ValueSetComponent } from './ValueSetComponent';
  */
 export class ValueSet extends FshEntity {
   id: string;
-  url?: string;
+  title?: string;
   description?: string;
-  inclusions: ValueSetComponent[];
-  exclusions: ValueSetComponent[];
+  components: ValueSetComponent[];
```

Comment:
I'm fine with this decision, but just for understanding, was this change made just because there are now far more components that exist in addition to just inclusions and exclusions?

Reply:
This change was made for ease of implementation, but I could honestly go either way on it, since the alternative isn't really much more complicated. It might be easier to deal with during export to have them separated in advance.

Reply:
This is fine to me, thanks for explaining!